Fedoraware/Fedoraware-TF2/src/Features/ESP/ESP.cpp

```diff
@@ -389,25 +389,25 @@ void CESP::DrawPlayers(CBaseEntity* pLocal)
 					{
 						case 0:
 						{
-							g_Draw.String(FONT, middle, y, Vars::Colours::Friend.Value, ALIGN_CENTERHORIZONTAL, "FRIEND");
+							g_Draw.String(FONT, middle, y - 28, Vars::Colours::Friend.Value, ALIGN_CENTERHORIZONTAL, "FRIEND");
```

Comment:
Where does the number 28 come from? I'm pretty sure it should be `y + nTextOffset` so it's font-size independant.